### PR TITLE
[FLOSS H3] docs: harmonize spelling of Git and CMake/cmake

### LINKED ISSUE #4679: Doc: harmonize Git spelling
As brought up in https://github.com/ElektraInitiative/libelektra/pull/4629#pullrequestreview-1172200219 Git is sometimes spelled as Git and sometimes as GIT.

As it is not an acronym - it should be changed to `Git`.

### PR DESCRIPTION
**Pull request by joni1993**
Fixes #4679 4679

## Basics

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation (see [Documentation Guidelines](https://www.libelektra.org/devgettingstarted/documentation))
- [ ] I fixed all affected decisions (see [Decision Process](https://www.libelektra.org/decisions/decision-process))
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://www.libelektra.org/devgettingstarted/coding))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

### CODE REVIEW

**Comment by markus2330** [comment]
Ahh, was it maybe by accident that doc/Git.md ended up in #4634?

As said over there: please make sure that doc/GIT.md gets a proper rename to doc/Git.md (e.g. by using `git mv`).

File: doc/TESTING.md [hunk 1]
```diff
@@ -393,9 +393,9 @@ following section show how the most common ones can be used with `libelektra`.
 file by calling it with `cppcheck --enable=all <sourcefile>`. This way it might miss some header
 files though and thus doesn't detect all possible issues, but still gives useful hints in general.
 
-To analyze the whole project, use it in conjunction with `cmake` by calling `cmake` with the parameter
-`-DCMAKE_EXPORT_COMPILE_COMMANDS=ON`. This way `cmake` creates a file called `compile_commands.json` in
-the build directory. Afterwards, call `cppcheck` with the cmake settings and store the output as xml:
+To analyze the whole project, use it in conjunction with `CMake` by calling `cmake` with the parameter
+`-DCMAKE_EXPORT_COMPILE_COMMANDS=ON`. This way `CMake` creates a file called `compile_commands.json` in
```
Comment (flo91) [inline doc/TESTING.md:397]: I'd only put `cmake` (the command, lowercase) in `code`-tags.
```suggestion
To analyze the whole project, use it in conjunction with CMake by calling `cmake` with the parameter
`-DCMAKE_EXPORT_COMPILE_COMMANDS=ON`. This way CMake creates a file called `compile_commands.json` in
```

File: doc/news/2016-11-22_0.8.19.md [hunk 2]
```diff
@@ -252,7 +252,7 @@ Changed files are:
 
 ## Portability
 
-Elektra should work on every system that has `cmake` and a `C/C++` compiler.
+Elektra should work on every system that has `CMake` and a `C/C++` compiler.
```
Comment (flo91) [inline doc/news/2016-11-22_0.8.19.md:255]:
```suggestion
Elektra should work on every system that has CMake and a C/C++ compiler.
```

**Review by flo91 — CHANGES_REQUESTED** [review]

**Comment by deoknats861** [comment]
### Review
**Current CI situation**
Cirrus CI failes due to broken links. As I could not reproduce this issue locally, I would recommend to rerun the check but I do not know if there is already a solution for that (see [issue 4669](https://github.com/ElektraInitiative/libelektra/issues/4669) for updates).
Regarding the Jenkins build fail I cannot see exactly why it is failing. There were some errors for generated man pages, maybe there is a problem with that. I could also imagine that this is an internal Jenkins issue exclusively.

**Documentation**
The PR title and the the changes made to the notes for the next release speak are clear and describe everything that was changed in this PR. 

**Examples are well chosen and understandable**
There were no examples added.

**Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)**
No code was added.

**APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)**
No API was added.

**Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)**
No code was added.

**Comment by deoknats861** [comment]
Update: The CI problems occurred due to https://github.com/ElektraInitiative/libelektra/issues/4752

You can fix it be rebasing with the current master and repushing to your branch. CI should then succeed :)

**Review by flo91 — APPROVED** [review]

**Comment by flo91** [comment]
jenkins build libelektra please

**Comment by markus2330** [comment]
Great job!

**Pull request merged**